packages/spatial/src/renderer/components/PostProcessingComponent.tsx

```diff
@@ -135,6 +137,9 @@ const PostProcessingReactor = (props: { entity: Entity; rendererEntity: Entity }
     // //always have the outline effect for the highlight selection
     const outlineEffect = new OutlineEffect(scene as Scene, camera.value as ArrayCamera, getState(HighlightState))
     outlineEffect.selectionLayer = ObjectLayers.HighlightEffect
+    outlineEffect.blur = true
```

Comment:
we should use the values in `HighlightState` instead of hardcoding these, and fix those to be the settings we want

Reply:
implemented